src/test/fuzz/fuzz.cpp

```diff
@@ -131,13 +133,14 @@ fuzz(std::string const& filename, el::Level logLevel,
 restart:
 {
     VirtualClock clock;
-    Application::pointer app1 = Application::create(clock, cfg1);
-    Application::pointer app2 = Application::create(clock, cfg2);
+    auto app1 = createTestApplication(clock, cfg1);
```

Comment:
I was expecting to start to see changes related to proper randomization both before creating configs and in the "main fuzzer loop", something (for now), like:
```
            srand(1);
            gRandomEngine.seed(1);
```

Reply:
Ah ok. Added.